tools/dgeni/processors/docs-private-filter.ts

```diff
@@ -19,6 +19,10 @@ const INTERNAL_METHODS = [
   'registerOnChange',
   'registerOnTouched',
   'setDisabledState',
+  'setDescribedByIds',
+  'registerOnValidatorChange',
+  'onContainerClick',
+  'shouldPlaceholderFloat',
```

Comment:
Why add these here instead of marking them with `@docs-private`?

Reply:
I was on the fence either way. The pattern seem to be doing this over `@docs-private` so I went this way. Plus this is on several different components so easier to do it in one place vs several.

Reply:
`setDescribedByIds`, `onContainerClick` and `shouldPlaceholderFloat` are all part of `MatFormFieldControl` which I do want to have an API doc, I think doing it this way might inadvertently hide them there too

Reply:
@mmalerba - Got it, I'll pull them out in that case.